package.json

```diff
@@ -72,7 +72,7 @@
     "react-grid-layout": "^0.18.2",
     "react-resizable": "^1.10.1",
     "react-virtualized": "^9.21.2",
-    "sql-formatter": "git+https://github.com/getredash/sql-formatter.git",
+    "sql-formatter": "^15.0.2",
```

Comment:
FYI the main reason we had a custom `sql-formatter` maintained in [getredash/sql-formatter](https://github.com/getredash/sql-formatter) is due to the lack of support to the parameters syntax (`{{}}`). Previously it would just add line breaks, breaking the query text, with this version it seems to throw an error when a query has them

![Screenshot 2024-01-22 at 09 53 51](https://github.com/getredash/redash/assets/3356951/296f9381-71ae-4ced-a076-1cd0a273f5cc)